index.html

```diff
@@ -5,9 +5,173 @@
+        <nav class="topnav">
+            <ul>
+                <li><a href="#home" class="current">Home</a></li>
+                <li><a href="#about">About</a></li>
+                <li><a href="#ingredients">Ingredients</a> </li>
+                <li><a href="#menu">Menu</a></li>
+                <li> <a href="#reviews">Reviews</a></li>
+                <li><a href="#reservations">Reservations</a></li>
+            </ul>
+        </nav>
+        </div>
+
+        <h1> 
+            the right ingredients <br> for the right food
+        </h1>
+        <p>
+            <img src="/images/divider-white.png" alt="White divider">
+        </p>
+        <div class="buttons"> 
+            <button class="btn-1">BOOK A TABLE</button>
+            <button class="btn-2">SEE THE MENU</button>
+        </div>
+    </section>
+
+    <div class="cook">
+        <div> 
+            <h2>Just the right food</h2> 
+            <img src="/images/divider-gold.png" alt="Golden divider">
+            <p>If you've been to one of our restaurants, you've seen - and tasted - what keeps our customers coming back for more. Perfect materials and freshly baked food, delicious Lamba cakes, muffins, and gourmet coffes make us hard to resist! Stop in today and check us out!</p>
+            <img id="about-cook" src="/images/about-cook.png" alt="Cook decorating a plate">
+        </div>
+        <div>
+            <img id="steak-plate" src="/images/about-dish.png" alt="Plate">
+        </div>
+    </div>
+
+    <div class="ingredients">
+        <div class="container">
+            <h2>Fine ingredients</h2>
+            <img src="/images/divider-white.png" alt="White divider">
+            <p>If you've been to one of our restaurants, you've seen - and tasted - what keeps our customers coming back for more. Perfect materials and freshly baked food, delicious Lamba cakes, muffins, and gourmet coffes make us hard to resist! Stop in today and check us out!</p>
+            <div class="small-container">
+                <img class="miniatures" src="/images/ingredients-grains.png" alt="Grains">
+                <img class="miniatures" src="/images/ingredients-spices.png" alt="Spices">
+                <img class="miniatures" src="/images/ingredients-bread.png" alt="Bread">
+            </div>
+        </div>
+    </div>
+
+    <div class="menu">
+        <div class="item">
+            <h2>Appetisers <br><img src="/images/divider-gold.png" alt="Golden divider"></h2> 
+              <h3>Tzatsiki &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; $3.99</h3>
```

Comment:
Ouch! That's a lot of blank spaces! I remember doing this too, many years ago.
The correct approach would be to have the name of the item and the price in separate elements, and position them accordingly. I'm struggling to decide what would be the 'correct' approach. You could for example, `position: absolute` the price to the right of the container (and leave the `<h3>` as it is), or you could use flex to push them apart.